Dockerfile.control-plane

```diff
@@ -13,3 +13,5 @@ ENTRYPOINT /usr/bin/control-plane-operator
 
 LABEL io.openshift.release.operator=true
 LABEL io.openshift.hypershift.control-plane-operator-subcommands=true
+LABEL io.openshift.hypershift.control-plane-operator-reconcile.cluster-machine-approver=true
+LABEL io.openshift.hypershift.control-plane-operator-reconcile.cluster-autoscaler=true
```

Comment:
Please also add this to the `Dockerfile` and `Dockerfile.fast` so it will also work when using those during development